readme.txt

```diff
@@ -179,6 +179,7 @@ Check out our extensive [knowledgebase](https://m.tri.be/18wm) for articles on u
 = [5.0.4] TBD =
 
 * Tweak - Added admin notice when editing an Events Calendar Pro recurring event having tickets in classic editor. [ET-949]
+* Fix - Detect ticket provider(s) by correctly excluding RSVP. [ET-953]
```

Comment:
```suggestion
* Fix - Exclude the "RSVP" ticket provider from the Providers list in the Block Editor for Tickets, since RSVPs have their own block already. [ET-953]
```